src/riscv-hybrid-integration.adoc

```diff
@@ -279,9 +279,20 @@ shown in xref:default-csrnames-added[xrefstyle=short].
 
 {cheri_default_ext_name} optionally allows <<MODESW>> to execute in debug mode.
 
-When entering debug mode, the core always enters {cheri_cap_mode_name}.
+When entering debug mode, whether the core enters {cheri_int_mode_name} or
+{cheri_cap_mode_name} is UNSPECIFIED.
 Implementations may optionally support switching CHERI execution mode by
-executing the <<MODESW>> from the program buffer.
+executing <<MODESW>> from the program buffer.
```

Comment:
I think one thing we should clarify here is whether this also updates dinfc.MODE and therefore whether it only affects the current program buffer execution or also future entries to debug mode. I think having it also set dinfc.mode is cleaner and allows a debugger to just execute a single modesw in the program buffer to have all future debug entries be in capmode.

Reply:
done